docs/user_guide/ttl.md

```diff
@@ -14,10 +14,34 @@ store-version, after the push finishes, but before swapping the read traffic ove
 replay phase where the last N seconds of buffered real-time data is replayed. The replayed data gets overlaid on top of
 the data from the Full Push.
 
-This can be leveraged in order to achieve a TTL behavior. By scheduling periodic empty pushes and configuring how far 
-back to replay the real-time data (N), one can control the TTL parameters. The time to live is defined by N, which acts 
-as a "minimum TTL", while the "maximum TTL" is N + delay between each empty push. For example, if you schedule a daily 
-empty push, and N = 6 days, then the oldest data in your store will be at least 6 days old, and at most 7 days old.
+This can be leveraged in order to achieve a TTL behavior. Currently, there are two options that leverages this mechanism. 
+
+### Empty push
+By scheduling periodic empty pushes and configuring how far back to replay the real-time data (N), one can control the 
+TTL parameters. The time to live is defined by N, which acts as a "minimum TTL", while the "maximum TTL" is N + delay 
+between each empty push. For example, if you schedule a daily empty push, and N = 6 days, then the oldest data in your 
+store will be at least 6 days old, and at most 7 days old.
+
+The N is the `hybrid-rewind-seconds`, that's a part of hybrid store configuration.
+
+### [Experimental] Repush with TTL
+Similarly to empty push, the repush with TTL can also be configured and scheduled periodically to achieve the TTL behavior.
+The major differences between this and empty push are original data source and how real-time buffer is replayed.
+
+|                 | Data origin            | Real-time buffer replay                                               |
+|-----------------|------------------------|-----------------------------------------------------------------------|
+| Empty push      | None                   | Replay real-time buffer with `hybrid-rewind-seconds` config           |
+| Repush with TTL | Existing version topic | Replay real-time buffer with `rewind.time.in.seconds.override` config |
```

Comment:
may be add another row with just repush

Reply:
I don't see much value of adding repush, given this table is mainly for comparing TTL methods. If I miss anything, please let me know!